src/idiomatic/foundations-api-design/predictable-api/naming-conventions/06-with.md

```diff
@@ -0,0 +1,31 @@
+---
+minutes: 5
+---
+
+# With
+
+Prefix for builder-style configuration.
+
+```rust
+{Vec, BTreeSet}::with_capacity
+PathBuf::with_extension
+PathBuf::with_file_name
+```
+
+<details>
+- Prefix for methods that set something internally.
```

Comment:
Internally? They return the new value, so I wouldn't say that the mutation is done internally.